lms/static/js/learner_dashboard/views/course_card_actions.js

```diff
@@ -0,0 +1,41 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'edx-ui-toolkit/js/utils/html-utils',
+            'text!../../../templates/learner_dashboard/course_card_actions.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             pageTpl
+         ) {
+            return Backbone.View.extend({
+                tpl: HtmlUtils.template(pageTpl),
+
+                events: {
+                    'click .enroll-button': 'handleEnroll'
+                },
+
+                initialize: function() {
+                    this.render();
+                },
+
+                render: function() {
+                    var filledTemplate = this.tpl(this.model.toJSON());
+                    HtmlUtils.setHtml(this.$el, filledTemplate);
+                },
+
+                handleEnroll: function(){
+                    //Enrollment click event handled here
+                }
+            });
+        }
+    );
+}).call(this, define || RequireJS.define);
```

Comment:
My argument is that because ```View Course``` is shown based on enrollment status then it should be handled by the enroll view. I would prefer to settle on the name of a file before it is added to the repo. instead of adding it and then discussing. I just think that actions is too vague a name, when really the main function that this view is going to handle is enrollment.